### PR TITLE
tables, images, and more links/refs

### PR DESCRIPTION
**Pull request by kwsutter**
Tables were cleaned up for Issue #75.  Also cleaned up Images.  More missing links and references.  And, some more misc cleanup.  Cursory review and approval requested.  Thank you!

### CODE REVIEW

File: specification/src/main/asciidoc/platform/RevisionHistory.adoc [hunk 1]
```diff
@@ -7,10 +7,10 @@
 
 * Java EE 8 requires Java SE 8.
 * Added
```
Comment (bshannon) [inline specification/src/main/asciidoc/platform/RevisionHistory.adoc]: The revision history should be cleared out and replaced for Jakarta EE 8.

**Comment by kwsutter** [comment]
> Generally looks fine. One comment below. Also, consider using this for footnotes:
> https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#section-footnotes
> https://asciidoctor.org/docs/user-manual/#user-footnotes

Thanks.  The footnote work will be covered by this Issue #77.  It was next on my list...

File: specification/src/main/asciidoc/platform/RevisionHistory.adoc (hunk 1 above)
Comment (kwsutter) [reply specification/src/main/asciidoc/platform/RevisionHistory.adoc]: I the Revision History does show the history before Jakarta...  I'd rather leave it.

Comment (bshannon) [reply specification/src/main/asciidoc/platform/RevisionHistory.adoc]: Didn't we remove all the acknowledgements for those versions?

As it is, the revision history is only for the lifetime of the Java EE 8 spec.
We clear out the revision history with each new version.  Seems like we
should do the same here, indicating how this version of this Jakarta EE
spec came to be.  The things that happened early in the Java EE 8 spec
development are irrelevant.

**Pull request merged**